.changeset/bitter-teeth-think.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/vite-plugin": patch
+---
+
+Add support for binding to named entrypoints in the same worker
```

Comment:
Do we also need a patch changeset for fixing the bug around `cloudflare:workers` not being treated as a builtin?

Reply:
This PR doesn't fix that. I have another one incoming.